src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortEngine.java

```diff
@@ -164,13 +165,13 @@ public ExtractCohortEngine(final String projectID,
         this.maxLocation = maxLocation;
 
         this.printDebugInformation = printDebugInformation;
-        this.vqsLodSNPThreshold = vqsLodSNPThreshold;
-        this.vqsLodINDELThreshold = vqsLodINDELThreshold;
-        this.VQSLODFilteringType = VQSLODFilteringType;
+        this.vqScoreSNPThreshold = vqScoreSNPThreshold;
+        this.vqScoreINDELThreshold = vqScoreINDELThreshold;
+        this.vqScoreFilteringType = vqScoreFilteringType;
         this.excludeFilteredSites = excludeFilteredSites;
 
-        this.filterSetInfoTableRef = VQSLODFilteringType.equals(ExtractCohort.VQSLODFilteringType.NONE) ? null : new TableReference(filterSetInfoTableName, SchemaUtils.YNG_FIELDS);
-        this.filterSetSiteTableRef = VQSLODFilteringType.equals(ExtractCohort.VQSLODFilteringType.NONE) ? null : new TableReference(filterSetSiteTableName, SchemaUtils.FILTER_SET_SITE_FIELDS);
+        this.filterSetSiteTableRef = vqScoreFilteringType.equals(ExtractCohort.VQScoreFilteringType.NONE) ? null : new TableReference(filterSetSiteTableName, SchemaUtils.FILTER_SET_SITE_FIELDS);
+        this.filterSetInfoTableRef = vqScoreFilteringType.equals(ExtractCohort.VQScoreFilteringType.NONE) ? null : new TableReference(filterSetInfoTableName, getFilterSetInfoTableFields());
```

Comment:
why a method instead of the constant?

Reply:
I have created a class 'ExtractCohortLiteEngine' which extends 'ExtractCohortEngine' and overrides specific methods. So when you call it in 'Classic' mode it uses the method call to get the constant for 'Classic' and when you call it in 'Lite' mode, it uses the overridden method to get the constant for 'Lite'.